lib/runners/testing/smoke.rb

```diff
@@ -13,6 +13,8 @@ class Smoke
 
       PROJECT_PATH = "/project".freeze
 
+      TestSet = Struct.new(:name, :pattern, :options, keyword_init: true)
```

Comment:
[suggestion] What about doing it simply like this?

```suggestion
      TestSet = Struct.new(:name, :pattern, :offline, keyword_init: true)
```